### PR TITLE
REF: navigationStyle default close position

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/rnpexll6jea4yyial7j2jmln6u?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1895)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/vpvbwcuva4pujrhdnx5pgplcyu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1896)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

File: components/navigationStyle.tsx [hunk 1]
```diff
@@ -23,32 +29,44 @@ export type NavigationOptionsGetter = (theme: Theme) => (deps: { navigation: any
...
-      // Determine if the current screen is the first one in the stack using the updated method
       const isFirstRouteInStack = navigation.getState().index === 0;
+      const isModal = route.params?.presentation !== 'card';
+
+      const closeButton =
+        closeButtonPosition !== undefined
+          ? closeButtonPosition
+          : isFirstRouteInStack && isModal
+            ? CloseButtonPosition.Right
+            : CloseButtonPosition.None;
```
Comment (Overtorment) [inline components/navigationStyle.tsx:52]: fuck this is hard to read

**Review by Overtorment — APPROVED** [review]
mostly lgtm, extept this nested ternary hell

File: components/navigationStyle.tsx (hunk 1 above)
Comment (marcosrdz) [reply components/navigationStyle.tsx]: refactored

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/qy3yxez4kb3lltav7o5htigoha?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1907)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**